openapi/bootstrap-agent.json

```diff
@@ -229,6 +229,51 @@
           }
         ]
       },
+      "BfdMode": {
```

Comment:
These API additions fall out from the addition of BGP configuration to `RackNetworkConfigV1`